roles/uperf/templates/server.yml.j2

```diff
@@ -22,6 +22,8 @@ spec:
 {% endif %}
 {% if workload_args.hostnetwork is sameas true %}
       hostNetwork: true
+      serviceAccountName: benchmark-operator
+      serviceAccount: benchmark-operator
```

Comment:
Quick question, do we need both `serviceAccountName` and `serviceAccount` ?

Reply:
nope, in fact serviceAccount is deprecated...
```
[root@ip-172-31-78-162 ~]# oc explain pod.spec.serviceAccount
KIND:     Pod
VERSION:  v1

FIELD:    serviceAccount <string>

DESCRIPTION:
     DeprecatedServiceAccount is a depreciated alias for ServiceAccountName.
     Deprecated: Use serviceAccountName instead.
```

Fixing it...